automatic/apache-httpd/tools/helpers.ps1

```diff
@@ -34,7 +34,7 @@ function Assert-TcpPortIsOpen {
         Select-Object @{Name = "Id"; Expression = {$_} } | `
         Get-Process | `
         Select-Object Name, Path
-
+$configFile
```

Comment:
What is this line doing here?

Reply:
I have absolutely no idea 😕 

Good catch. Let me fix that ...